lib/record_store/provider/dynect.rb

```diff
@@ -82,16 +82,28 @@ def secrets
         super.fetch('dynect')
       end
 
+      def api_rdata(record)
+        case record.type
+        when 'TXT'
+          { txtdata: record.rdata_txt }
+        else
+          record.rdata
+        end
+      end
+
       def build_from_api(api_record)
-        record = api_record.merge(api_record.fetch('rdata')).slice!('rdata').symbolize_keys
+        rdata = api_record.fetch('rdata')
+        record = api_record.merge(rdata).slice!('rdata').symbolize_keys
 
-        return if record.fetch(:record_type) == 'SOA'
+        type = record.fetch(:record_type)
+        return if type == 'SOA'
 
-        unless record.fetch(:fqdn).ends_with?('.')
-          record[:fqdn] = "#{record.fetch(:fqdn)}."
-        end
+        record[:txtdata].gsub!('\"', '"') if type == 'TXT'
```

Comment:
Just `TXT`? Do we need to also include SPF, just like we do for Google and DNSimple?